Dutch translation of the strings

nvbln opened the pull request:
Note: I have not compiled the files to test if there weren't any errors.

manuroe commented:
Thanks @nvbln.

Riot also uses translations strings from MatrixKit (https://github.com/matrix-org/matrix-ios-kit/blob/master/MatrixKit/Assets/MatrixKitAssets.bundle/en.lproj/MatrixKit.strings). May I ask you to translate them too?

manuroe commented:
@nvbln, as per https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst, can you please sign off this PR?

manuroe commented on Riot/Assets/nl.lproj/InfoPlist.strings:
```diff
@@ -0,0 +1,21 @@
+/*
+ Copyright 2016 OpenMarket Ltd
```
Our copyright is now *Copyright 2017 Vector Creations Ltd*

manuroe commented on Riot/Assets/nl.lproj/Localizable.strings:
```diff
@@ -0,0 +1,99 @@
+/*
+ Copyright 2015 OpenMarket Ltd
```
Our copyright is now *Copyright 2017 Vector Creations Ltd*

manuroe commented on Riot/Assets/nl.lproj/Vector.strings:
```diff
@@ -0,0 +1,475 @@
+/*
+ Copyright 2015 OpenMarket Ltd
```
Our copyright is now *Copyright 2017 Vector Creations Ltd*

nvbln commented:
@manuroe 
> May I ask you to translate them too?

Thanks for pointing that out. I will once I have a good few hours of free time, I am afraid that isn't now though. 

As per request I changed the copyright dates (and added myself to the authors list) in the new commit. 

> can you please sign off this PR?

Of course. I hope you don't mind it if I don't post my email in plain text though (although I am sure it is bound to turn up somewhere), you can always see it in the git commits.

Signed-off-by: Nathan van Beelen

manuroe commented:
@nvbln, your PR is approved.
I will merge it once we are done with our current release candidate. So, your translations will be not in the coming release but the one after. Hopefully, translations on MatrixKit will be available at that time for a full translation of the app.
